### PR TITLE
Switch to Material 3

### PR DESCRIPTION
**Pull request by crisbeto**
Reworks the docs site to use the latest Material Design spec. Also switches to the new Angular logo and fixes some general UI inconsistencies.

image
image

### CODE REVIEW

File: src/app/pages/component-page-header/component-page-header.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <header class="docs-primary-header component-page-header">
+  <!-- TODO: this needs more work. Probably some text. -->
```
[the remark this answers is not in the document]
Comment (crisbeto) [reply src/app/pages/component-page-header/component-page-header.html:2]: Ah, I left this for myself to fix before I send out the PR, but I missed it when I was looking through the diff. I'll fix it now.

**Comment by crisbeto** [comment]
> Should DevRel sign off on how the logo appears? Like...multicolored or always black/white?

I'll ping devrel on it. I think the multi-colored logo is correct, but in general the site would need a redesign to fit better with the new brand.

> I think it'd be more "M3" to have a non-colored header

Agreed, but I think we'd have to use something else to add color to the hero section. For now I'm just keeping the old design and making it work with M3.

> These themes look nice, would you propose having this be our new prebuilt themes?

Yeah, I was thinking of using something similar for the prebuilts in the main repo.

> Off-topic: Are we doing state layers right? They look so muted/grey

I _think_ so, here's an example from the spec:
image

**Review by andrewseguin — APPROVED** [review]
All sounds good, this is good to go

**Comment by crisbeto** [comment]
Devrel confirmed that this is fine for now. Merging.

**Pull request merged**